bigtable/doc/Doxyfile

```diff
@@ -119,7 +119,7 @@ WARN_LOGFILE           =
 # Configuration options related to the input files
 #---------------------------------------------------------------------------
 
-INPUT                  = README.md doc client
+INPUT                  = README.md doc client admin
```

Comment:
Technically, both Data API and Admin API interfaces are "clients". While earlier, having just one client (for data) meant that `client` = "data client" and that was OK, now that you're adding an admin client, having `client` = "data client" while `admin` = "admin client" is a bit confusing.

Can we make this explicit and have a `data_client` and `admin_client`?

Reply:
Created #103 to track this.  I would prefer not to make another big change in this PR.